sample-app/src/main.js

```diff
@@ -30,6 +31,9 @@ const questions1 = [
   {
     type: "password",
     guiType: "login",
```

Comment:
If `guiType` is deprecated, let's not show it in the sample-app. We should still test for it, until no longer supported.